### PR TITLE
html5-form

### PR DESCRIPTION
**Pull request by o-msh**
### html5-form

[Demo](https://o-msh.github.io/kottans-frontend/practice/form/dist/) |
[Code base](https://github.com/o-msh/kottans-frontend/tree/master/practice/form/src)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/o-msh/html5-form/index.html [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+                                <h3 class="form-help__title"><span>help center</span></h3>
+                            </div>
+                            <div class="form-help__text">
+                                <div class="form-help__about">
+                                    If you need assistance while placing your order, contact one of our beloved Project
+                                    Manager
+                                    by phone or email.
+                                </div>
+                                <address class="form-help__address">
+                                    <p><a href="#">+971 (0) 555 555</a></p>
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/html5-form/index.html]: use the same way as for email link. [Read more](https://developer.mozilla.org/ru/docs/Web/HTML/Element/A#%D0%A1%D0%BE%D0%B7%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5_%D1%81%D1%81%D1%8B%D0%BB%D0%BA%D0%B8_%D1%81_%D0%BD%D0%BE%D0%BC%D0%B5%D1%80%D0%BE%D0%BC_%D1%82%D0%B5%D0%BB%D0%B5%D1%84%D0%BE%D0%BD%D0%B0)

File: submissions/o-msh/html5-form/index.html [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="./css/style.css">
+    <link href="./css/font-awesome.css" rel="stylesheet">
+    <title>Form</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <button class="header__button header__button_back"><span>go back</span></button>
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/html5-form/index.html]: It looks like a button but functionally it's a link.

File: submissions/o-msh/html5-form/index.html [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+                        </div>
+                    </section>
+                    <section class="form__languages languages-section">
+                        <h3 class="languages-section__title">choose your languages pairings</h3>
+                        <div class="languages-section__container languages-container">
+                            <div class="languages-container__from container-from">
+                                <div class="container-from__wrapper select-wrapper">
+                                    <label class="select-wrapper__label">
+                                        <div class="select-wrapper__title">from</div>
+                                        <select name="language-from" class="select-wrapper__select">
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/html5-form/index.html]: This input is required.

File: submissions/o-msh/html5-form/index.html [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+                                            <option value="poland" class="select-wrapper__option">poland</option>
+                                        </select>
+                                    </label>
+                                </div>
+                            </div>
+                            <div class="languages-container__to container-to">
+                                <div class="container-to__wrapper select-wrapper">
+                                    <label class="select-wrapper__label">
+                                        <div class="select-wrapper__title">to</div>
+                                        <select name="language-to" class="select-wrapper__select">
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/html5-form/index.html]: This input is required too.

File: submissions/o-msh/html5-form/index.html [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+                                        </select>
+                                    </label>
+                                    <label class="select-wrapper__label select-wrapper__label_to">
+                                        <select name="language-to" class="select-wrapper__select">
+                                            <option value="english">kurdish</option>
+                                            <option value="ukrainian">ukrainian</option>
+                                            <option value="poland">poland</option>
+                                        </select>
+                                    </label>
+                                    <button class="select-wrapper__btn select-wrapper__btn_delete" type="button"></button>
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/html5-form/index.html]: Move a trash button closer to the corresponding `select`.

File: submissions/o-msh/html5-form/index.html [hunk 6]
```diff
@@ -0,0 +1,366 @@
...
+                        <h3 class="payment-section__title">sign up and choose a payment method</h3>
+                        <div class="payment-section__sign-in sign-in">
+                            <div class="sign-in__text">
+                                <div class="sign-in__additional-text">already have an account with us?</div>
+                                <div class="sign-in__link"><a href="#">sign in.</a></div>
+                            </div>
+                        </div>
+                        <div class="payment-section__user-information user-information">
+                            <label class="user-information__label">
+                                <div class="user-information__text">full name</div>
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/html5-form/index.html]: This text must be a `label`.

File: submissions/o-msh/html5-form/index.html [hunk 7]
```diff
@@ -0,0 +1,366 @@
...
+                        and social media accounts translations.
+                    </div>
+                </div>
+            </div>
+            <div class="footer__block footer__block_address">
+                <div class="footer__content">
+                    <h3 class="footer__title footer__title_address">our main office</h3>
+                    <div class="footer__text">
+                        <address class="footer__address">
+                            Dubai, UAE<br>P.O.Box: 502735<br>Phone: +971 (0) 44278128
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/html5-form/index.html]: Avoid using <br> tag for formatting text, use CSS for this.

File: submissions/o-msh/html5-form/index.html (hunk 1 above)
Comment (o-msh) [reply submissions/o-msh/html5-form/index.html]: Thank for advice :+1:

File: submissions/o-msh/html5-form/index.html (hunk 2 above)
Comment (o-msh) [reply submissions/o-msh/html5-form/index.html]: Exactly, i fixed.

File: submissions/o-msh/html5-form/index.html (hunk 3 above)
Comment (o-msh) [reply submissions/o-msh/html5-form/index.html]: Added `required`

File: submissions/o-msh/html5-form/index.html (hunk 7 above)
Comment (o-msh) [reply submissions/o-msh/html5-form/index.html]: Of course, thank you.

**Comment by o-msh** [comment]
@A-Ostrovnyy Thank for review, can you watch again, please?

**Comment by A-Ostrovnyy** [comment]
Well Done!

**Pull request merged**